src/background/adblocker.js

```diff
@@ -33,6 +33,8 @@ import Config, {
 
 let options = Options;
 
+const REDIRECT_RESOURCE_PATH_PREFIX = 'rule_resources/redirects/';
```

Comment:
This constant is only used once. Should I inline it?

Reply:
You can, because the usage context is enough for understanding.

Reply:
inlined in 38b81a2b737adcb9a5afa355b0e8c341783f4174